pipelines/batch/src/main/java/com/google/fhir/analytics/FhirSearchUtil.java

```diff
@@ -317,6 +332,88 @@ Bundle searchByPatientAndLastDate(
     return query.execute();
   }
 
+  /**
+   * Fetch the details of bulk export job for the given bulkExportStatusUrl
+   *
+   * @param bulkExportStatusUrl the url of the bulk export job
+   * @return BulkExportHttpResponse - the status and details of the bulk export job
+   * @throws IOException
+   */
+  public BulkExportHttpResponse fetchBulkExportHttpResponse(String bulkExportStatusUrl)
```

Comment:
Suggestion: Move this and the next method to a separate class as these are not "search" related. I think all you need is the `fetchUtil` which you can still have in a separate class too (is there anything else shared?). If you want to keep these here, consider renaming the class.